frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -156,7 +164,60 @@ export default RpConnectPipelinesCreate;
 export const PipelineEditor = observer((p: {
     yaml: string,
     onChange: (newYaml: string) => void
+    secrets?: string[]
 }) => {
+    const secrets = p.secrets ?? [];
+    // add custom autocomplete for ${secrets.<SECRET_ID>}
+    const addCustomAutocomplete = (monaco: Monaco) => {
+        monaco.languages.registerCompletionItemProvider('yaml', {
+            // Display suggestions for lines beginning with ${ or . to assist with variable interpolation
+            triggerCharacters: ['${', '.'],
+            provideCompletionItems(model: editor.ITextModel, position: Position): languages.ProviderResult<languages.CompletionList> {
+                const wordInfo = model.getWordUntilPosition(position);
+                const previousInfo = model.getWordUntilPosition({lineNumber: position.lineNumber, column: wordInfo.startColumn - 1});
+                const range = {
+                    startLineNumber: position.lineNumber,
+                    endLineNumber: position.lineNumber,
```

Comment:
It's always going to start and end on the same line, is that intentional?

Reply:
yes, it is intencional, but I disabled autocomplete, because it needs more testing and there isn't time now

Reply:
remove this function for now, I need to improve the autocomplete